File: Visual Studio Product Team/Visual Studio 2013 SDK Samples/115802-Visual Studio 2013 SDK Samples/Spell Checker - VS 2013/VB/Spelling/IMisspellingTag.vb
Imports System.Text
Imports Microsoft.VisualStudio.Text.Tagging

Namespace Microsoft.VisualStudio.Language.Spellchecker
	Friend Interface IMisspellingTag
		Inherits ITag
		ReadOnly Property Suggestions() As IEnumerable(Of String)
	End Interface
End Namespace

File: Visual Studio Product Team/Visual Studio 2010 SDK Samples/[C#]-Visual Studio 2010 SDK Samples/VS2010 Spell Checker/C#/Spelling/IMisspellingTag.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Microsoft.VisualStudio.Text.Tagging;

namespace Microsoft.VisualStudio.Language.Spellchecker
{
    interface IMisspellingTag : ITag
    {
        IEnumerable<string> Suggestions { get; }
    }
}
